contrib/pg_tde/t/pgtde.pm

```diff
@@ -81,4 +81,58 @@ sub compare_results
 	return compare($expected_filename_with_path, $out_filename_with_path);
 }
 
+# Common TDE helpers
+
+# Check if the encryption status of a table is as expected and return 't' or 'f'
+sub check_encryption_status {
+	my ($node, $table_name, $expected) = @_;
+	my $result = safe_psql('postgres', "SELECT pg_tde_is_encrypted('$table_name')");
+	append_to_result_file($node->name . ": encryption check result for $table_name = $result");
+	is($result, $expected, "Check encryption status for '$table_name' on " . $node->name);
```

Comment:
I suggest not having such a test assertion (anything we use from Test::More) as part of a helper function; this should be handled in the main test file. That will make it easier to understand and follow the test case.